src/messaging/messaging.ts

```diff
@@ -134,9 +134,9 @@ function mapRawResponseToDevicesResponse(response: object): MessagingDevicesResp
 /**
  * Maps a raw FCM server response to a MessagingDeviceGroupResponse object.
  *
- * @param {object} response The raw FCM server response to map.
```

Comment:
Are these values in the curly brackets types?  Not sure if they are important enough to try and transmit to the new format.

Reply:
This is the old jsdoc way of documenting types. That's not necessary with TypeScript and Typedoc/tsdoc.